SystemInfo/Collector/JsonComposerLockSystemInfoCollector.php

```diff
@@ -74,6 +75,14 @@ public function collect()
         $lockData = json_decode(file_get_contents($this->lockFile), true);
         $jsonData = json_decode(file_get_contents($this->jsonFile), true);
 
+        if (!$lockData) {
```

Comment:
```suggestion
        if ($lockData === null) {
```

> null is returned if the json cannot be decoded or if the encoded data is deeper than the nesting limit.

ref. https://www.php.net/manual/en/function.json-decode 

or

```suggestion
        if (!is_array($lockData)) {
```
as this is what `\EzSystems\EzSupportToolsBundle\SystemInfo\Collector\JsonComposerLockSystemInfoCollector::{extractPackages, extractRepositoryUrls}` expects as input